resources/extruders/bondtech_bmgx2m_extruder_0.def.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": 2,
+    "name": "Left Extruder",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "smoothie_dual",
+        "position": "0"
+    },
+
+    "overrides": {
+        "extruder_nr": {
+            "default_value": 0,
+            "maximum_value": "1"
+        },
+        "machine_nozzle_offset_x": { "default_value": 0 },
+        "machine_nozzle_offset_y": { "default_value": 0 },
+        "machine_nozzle_size": { "default_value": 0.8 },
+        "material_diameter": { "default_value": 1.75 }
```

Comment:
Similarly, I don't think that the Smoothieboard makes your nozzle 0.8mm or have 1.75mm filament.